### PR TITLE
Add a player method to the client

### LINKED ISSUE #2: Can I retrieve Player information from NemeStats?

### PR DESCRIPTION
**Pull request by ericworkman**
Closes #2 probably. Removed the login-specific fields from the nemestats module and fixed the namespacing issue I showed you this morning. I'll add some documentation later.

Here's my player in our group:

```
NemeStatsClient.get_stats(:player, 13392, 16377)
[averagePlayersPerGame: 5.96923065, baseNemePoints: 683,
 gameDefinitionTotals: %{"summariesOfGameDefinitionTotals" => [%{"gameDefinitionId" => 34357,
      "gameDefinitionName" => "Mascarade (2013)", "gamesLost" => 14,
      "gamesWon" => 5},
    %{"gameDefinitionId" => 60223,
      "gameDefinitionName" => "One Night Ultimate Werewolf", "gamesLost" => 6,
      "gamesWon" => 3},
    %{"gameDefinitionId" => 67050, "gameDefinitionName" => "Codenames",
      "gamesLost" => 4, "gamesWon" => 3},
    %{"gameDefinitionId" => 73041,
      "gameDefinitionName" => "The Resistance: Avalon", "gamesLost" => 0,
      "gamesWon" => 2},
    %{"gameDefinitionId" => 39693,
      "gameDefinitionName" => "The Resistance (2009)", "gamesLost" => 3,
      "gamesWon" => 1},
    %{"gameDefinitionId" => 34355, "gameDefinitionName" => "Spyfall (2014)",
      "gamesLost" => 2, "gamesWon" => 1},
    %{"gameDefinitionId" => 67851, "gameDefinitionName" => "Hunters of Arcfall",
      "gamesLost" => 1, "gamesWon" => 1},
    %{"gameDefinitionId" => 68208, "gameDefinitionName" => "Salem",
      "gamesLost" => 0, "gamesWon" => 1},
    %{"gameDefinitionId" => 39056, "gameDefinitionName" => "Sushi Go! (2013)",
      "gamesLost" => 0, "gamesWon" => 1},
    %{"gameDefinitionId" => 38241, "gameDefinitionName" => "Zombie Dice (2010)",
      "gamesLost" => 0, "gamesWon" => 1},
    %{"gameDefinitionId" => 38647, "gameDefinitionName" => "Tsuro (2004)",
      "gamesLost" => 6, "gamesWon" => 0},
    %{"gameDefinitionId" => 34354,
      "gameDefinitionName" => "Exploding Kittens: NSFW Deck (2015)",
      "gamesLost" => 2, "gamesWon" => 0},
    %{"gameDefinitionId" => 34174,
      "gameDefinitionName" => "Small World: Cursed! (2009)", "gamesLost" => 2,
      "gamesWon" => 0},
    %{"gameDefinitionId" => 65496, "gameDefinitionName" => "BANG!",
      "gamesLost" => 1, "gamesWon" => 0},
    %{"gameDefinitionId" => 34637,
      "gameDefinitionName" => "Boss Monster: The Dungeon Building Card Game (2013)",
      "gamesLost" => 1, "gamesWon" => 0},
    %{"gameDefinitionId" => 34633,
      "gameDefinitionName" => "Letters to Santa (2012)", "gamesLost" => 1,
      "gamesWon" => 0},
    %{"gameDefinitionId" => 74308, "gameDefinitionName" => "Love Letter",
      "gamesLost" => 1, "gamesWon" => 0},
    %{"gameDefinitionId" => 71291, "gameDefinitionName" => "Saboteur (2004)",
      "gamesLost" => 1, "gamesWon" => 0},
    %{"gameDefinitionId" => 73003,
      "gameDefinitionName" => "The Resistance: Hostile Intent",
      "gamesLost" => 1, "gamesWon" => 0}]}, gameDurationBonusNemePoints: -126,
 totalGames: 65, totalGamesLost: 46, totalGamesWon: 19, totalPoints: 557,
 weightBonusNemePoints: 0, winPercentage: 29]
```

@Serneum

### CODE REVIEW

File: lib/nemestats_client.ex [hunk 1]
```diff
@@ -9,4 +9,8 @@ defmodule NemeStatsClient do
     body = Poison.encode!(%{userName: username, password: password})
     NemeStats.post("UserSessions/", [body: body]).body
   end
+
+  def player(gaming_group, player_id) do
```
Comment (Serneum) [inline lib/nemestats_client.ex]: Thoughts on making this `get_player`?

Comment (ericworkman) [reply lib/nemestats_client.ex]: Based on some conversation, how about `get_stats(:player, gaming_group_id, player_id)`?

Comment (Serneum) [reply lib/nemestats_client.ex]: Sounds good to me

Comment (ericworkman) [reply lib/nemestats_client.ex]: @Serneum Done

File: lib/nemestats_client.ex [hunk 2]
```diff
@@ -9,4 +9,9 @@ defmodule NemeStatsClient do
     body = Poison.encode!(%{userName: username, password: password})
     NemeStats.post("UserSessions/", [body: body]).body
   end
+
+  def get_stats(:player, gaming_group, player_id) do
+    NemeStats.get("GamingGroups/#{gaming_group}/PlayerStats/#{player_id}/").body
+  end
+  def get_stats(_, _, _), do: nil
```
Comment (Serneum) [inline lib/nemestats_client.ex:17]: Newline before def

**Review by Serneum — CHANGES_REQUESTED** [review]

File: lib/nemestats_client.ex (hunk 2 above)
Comment (ericworkman) [reply lib/nemestats_client.ex:17]: Done

**Review by Serneum — APPROVED** [review]

**Pull request merged**